stdlib/private/SwiftPrivatePthreadExtras/SwiftPrivatePthreadExtras.swift

```diff
@@ -53,8 +53,8 @@ internal func invokeBlockContext(
 ) -> UnsafeMutablePointer<Void>! {
   // The context is passed in +1; we're responsible for releasing it.
   let contextAsOpaque = OpaquePointer(contextAsVoidPointer!)
-  let context = Unmanaged<PthreadBlockContext>.fromOpaque(contextAsOpaque)
-    .takeRetainedValue()
+  let context = Unmanaged<PthreadBlockContext>.fromOpaque(
+    UnsafePointer(contextAsOpaque)).takeRetainedValue()
```

Comment:
I think you can simplify this by using `.fromOpaque(contextAsVoidPointer!)`, and eliminating the `contextAsOpaque` variable.